docs/changelog/104481.yaml

```diff
@@ -0,0 +1,6 @@
+pr: 104481
```

Comment:
Shouldn't this be the number of the current PR (which is `104486`) or is it the number of the original PR for backports?

Reply:
For backports it's the number of the original PR.